templates/settings.yaml.erb

```diff
@@ -42,8 +42,16 @@
 <% scope.lookupvar('foreman::cors_domains').each do |domain| -%>
  - '<%= domain %>'
 <% end -%>
+<% end -%>
 
```

Comment:
Please keep the old style line endings and also apply it to `trusted_proxies`. This helps avoid empty lines. with your change here you end up with:
```yaml
:logging:
  # ...


 # Individual logging types can be toggled on/off here
:loggers:
```

The previous code left out that empty line in the resulting file. That's why the empty line was within the `unless` block.

Reply:
hopefully closed.